Configurations.md

```diff
@@ -389,6 +389,255 @@ fn example() {
 }
 ```
 
+## `chains_block_parent_indent_children`
```

Comment:
I always have such a hard time coming up with config names 😄 If this approach seems viable for handling the scenario in #3157 (and it should indeed be configurable) then this should probably be renamed to something more succinct